Lib/fontmake/font_project.py

```diff
@@ -541,6 +540,12 @@ def save_otfs(
                 if "GSUB" in gsub_src:
                     font["GSUB"] = gsub_src["GSUB"]
 
+            # Read autohinting parameters from ufo lib if present
+            if "com.schriftgestaltung.customParameter.InstanceDescriptorAsGSInstance.TTFAutohint options" in ufo.lib and autohint is None:
+                autohint = ufo.lib["com.schriftgestaltung.customParameter.InstanceDescriptorAsGSInstance.TTFAutohint options"]
```

Comment:
you should move this lib key name at the top of the module next to similar constants.
It's pretty horrible name by the way, but I suppose that's what glyphsLib spits out.. /cc @belluzj